.golangci.next.reference.yml

```diff
@@ -2386,6 +2386,12 @@ linters-settings:
     # Default: false
     enable-all-rules: true
 
+    # Enable validation of comment directives
+    # See https://github.com/mgechev/revive#comment-directives
+    directives:
+      - name: specify-disable-reason
```

Comment:
We can mention that the `severity` is `warning` by default.

Reply:
Something like: https://github.com/golangci/golangci-lint/pull/5448#discussion_r1962143315

Reply:
we don't care about the severity, the golangci-lint is not the revive doc.

Reply:
Just came here to ask for the same thing. All other settings have their default value listed, this one doesn't. +1 for mentioning it here as well, even if it's a default value of a different project.